mappings/net/minecraft/client/network/ClientPlayerEntity.mapping

```diff
@@ -93,6 +93,8 @@ CLASS net/minecraft/unmapped/C_jlopmfei net/minecraft/client/network/ClientPlaye
 	METHOD m_sliqpvmd canSprint ()Z
 	METHOD m_spcgxwbg setClientPermissionLevel (I)V
 		ARG 1 clientPermissionLevel
+	METHOD m_syghdsvg onGameModeChanged (Lnet/minecraft/unmapped/C_lghcpyvl;)V
```

Comment:
Not so much an issue with this in particular, but the tense of `onOccurrence` event-handler-like methods seems inconsistent afaict.

Why is this past tense but `ClientPlayerEntity#onPickupSlotClick` is not?

This seems like something that should be in CONVENTIONS.md, either clarifying that they should all be one way, or clarifying when to use one way or the other.

Reply:
I'm definitely in favor of past tense here, I agree it should be clarified